dev-aws/kafka-shared/iam/iam.tf

```diff
@@ -233,3 +233,28 @@ resource "kafka_topic" "iam_revoked_v1" {
     "cleanup.policy"    = "delete"
   }
 }
+
+
+resource "kafka_topic" "iam_credentials_public_events_mapper_v1" {
+    name = "auth-customer.credentials-public-events-mapper-v1"
```

Comment:
Topic name for the private queue is `auth-customer.iam-credentials-v1`, so am thinking something like `auth-customer.iam-credentials-v1-public` or something similar would be more appropriate. Wdyt? 🤔